cfgrammar/src/lib/header.rs

```diff
@@ -0,0 +1,186 @@
+use crate::Span;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+#[derive(Debug)]
+#[non_exhaustive]
+pub enum HeaderError {
+    MissingGrmtoolsSection(Span),
+    IllegalName(Span),
+    InvalidValue(Span),
+    MalformedPath(Span),
+    ExpectedToken(char, Span),
+}
+
+#[derive(Debug, Eq, PartialEq)]
+pub enum ValBind<'a> {
+    // ! [A-Za-z]+
```

Comment:
Probably don't need these comments.

Reply:
Should be fixed in [a56b217](https://github.com/softdevteam/grmtools/pull/544/commits/a56b21779e29b99d3778c4b5afaf0e6f6287cbd5)